docs/source/deployment/lambda.mdx

```diff
@@ -281,7 +286,7 @@ If you ever want to remove the S3 bucket or Lambda functions that Serverless cre
 
 ## Customizing HTTP behavior
 
-The `@as-integrations/aws-lambda` package is compatible with Lambda's API Gateway V1 (REST) and V2 (HTTP). However, it does not let you customize HTTP behavior directly or support other AWS products that invoke Lambda functions (e.g., Application Load Balancers).
+The `@as-integrations/aws-lambda` package is compatible with the following event types `APIGatewayProxyEvent`, `APIGatewayProxyEventV2`, `ALBEvent`. This supports a wide range of eservices like  However, it does not let you customize HTTP behavior directly or support other AWS products that invoke Lambda functions (e.g., S3 or DynamoDB).
```

Comment:
The spellcheck CI is actually for once detecting a real typo. Also it looks like you inserted a half-sentence in this middle here.